rio_tiler/io/__init__.py

```diff
@@ -1,5 +1,8 @@
 """rio-tiler.io"""
 
 from .base import BaseReader, MultiBandReader, MultiBaseReader  # noqa
-from .cogeo import COGReader  # noqa
+from .rasterio import Reader
 from .stac import STACReader  # noqa
+
+# Keep Compatibility with <4.0
+COGReader = Reader
```

Comment:
I believe most user already do `from rio_tiler.io import COGReader` not `from rio_tiler.io.cogeo import COGreader`